Stoica Tudor/problema 3.4/problema 3.4/main.c

```diff
@@ -0,0 +1,91 @@
+#include <stdlib.h>
+#include "functii_elementare.h"
+
+char s1[1005],s2[1005];
+int v1[1005],size1;
+int v2[1005],size2;
+int maxx,dif,carry;
+int rez[1005],size;
+bool chooseOperation;
+
+int main()
+{
+    coutMessage("Choose you operation (0-Addition/1-Multiplication)");
+    cinInt(&chooseOperation);
+
+    if(chooseOperation==0) /// addition
+    {
+        size1=cinInt();
+        cinNextCharVec(s1,size1);
+
+        size2=cinInt();
+        cinNextCharVec(s2,size2);
+
+        maxx=max(size1,size2);
+        dif=maxx-min(size1,size2);
+
+        for(int i=1; i<=maxx; i++)
```

Comment:
This whole thing should be made into a function, not put all here.

And again, for the love of God, start from 0 not from 1